### PR TITLE
Quick fixes

### LINKED ISSUE #666: Turquoise Tag Color bugged
Example:
image

### LINKED ISSUE #674: Recently Updated Cubes display bug
See:
image

This happens after this section is expanded, then collapsed again. The label needs to be fixed.

### LINKED ISSUE #675: Follow Cube Button Visible when logged out
This button needs to not be visible

### PR DESCRIPTION
**Pull request by anderoonies**
fixes #675
fixes #674
fixes #666 😈 

## No "Follow" button when logged out
image

## Fix toggling
cubeexpand

## Turqouise
image

### CODE REVIEW

File: src/cube_overview.js [hunk 1]
```diff
@@ -9,107 +9,131 @@ import CSRFForm from './components/CSRFForm';
...
+                    Unfollow
+                  </Button>
+                ) : (
+                  <Button color="success" className="rounded-0" onClick={this.follow}>
+                    Follow
+                  </Button>
+                )
+              ) : (
+                []
+              )}
```
Comment (anderoonies) [inline src/cube_overview.js]: this is the only meaningful change in this file, the rest is eslint going to town

File: views/index.pug [hunk 2]
```diff
@@ -64,15 +64,15 @@ block content
...
+													a(href='/user/view/'+cube.owner) #{cube.owner_name}
 											a(href='/cube/overview/'+get_cube_id(cube))
 												img.card-img-top(src=cube.image_uri style='width: 100%;height: auto;')
-											em.text-right(style='padding: 2px;') Art by: #{cube.image_artist}                      
+											em.text-right(style='padding: 2px;') Art by: #{cube.image_artist}
 											a.btn.btn-success.rounded-0(href='/cube/overview/'+get_cube_id(cube)) View
 										br
 				.card-footer
-					a#recentMore(data-toggle="collapse" href="#recentCollapse" onclick="toggleRecent()") View More..
+					a#recentMore(data-toggle="collapse" href="#recentCollapse" onclick="toggleRecent()") View More...
```
Comment (anderoonies) [inline views/index.pug]: meaningful change #1

File: views/index.pug [hunk 3]
```diff
@@ -119,15 +119,15 @@ block content
...
+													a(href='/user/view/'+cube.owner) #{cube.owner_name}
 											a(href='/cube/overview/'+get_cube_id(cube))
 												img.card-img-top(src=cube.image_uri style='width: 100%;height: auto;')
-											em.text-right(style='padding: 2px;') Art by: #{cube.image_artist}                      
+											em.text-right(style='padding: 2px;') Art by: #{cube.image_artist}
 											a.btn.btn-success.rounded-0(href='/cube/overview/'+get_cube_id(cube)) View
 										br
 				.card-footer
-					a#draftMore(data-toggle="collapse" href="#draftCollapse" onclick="toggleDraft()") View More..
+					a#draftMore(data-toggle="collapse" href="#draftCollapse" onclick="toggleDraft()") View More...
```
Comment (anderoonies) [inline views/index.pug]: meaningful change #2

**Comment by dekkerglen** [comment]
Can you run npm run-script beautify and push ?

**Comment by anderoonies** [comment]
@dekkerglen beautified

**Pull request merged**